src/Engines/MeiliSearchEngine.php

```diff
@@ -126,6 +126,12 @@ protected function performSearch(Builder $builder, array $searchParams = [])
     {
         $meilisearch = $this->meilisearch->index($builder->index ?: $builder->model->searchableAs());
 
+        // from 0.21.0 onwards, `filters` is renamed to `filter`
+        if (version_compare($this->getInstalledMeilisearchVersion(), '0.21.0') >= 0) {
```

Comment:
```suggestion
        if (version_compare(MeiliSearch::VERSION, '0.21.0') >= 0) {
```

And rename the current `MeiliSearch` alias to `MeiliSearchClient`.